src/main/flight/pid.c

```diff
@@ -426,7 +426,7 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_
 #endif
     angle = constrainf(angle, -pidProfile->levelAngleLimit, pidProfile->levelAngleLimit);
     const float errorAngle = angle - ((attitude.raw[axis] - angleTrim->raw[axis]) / 10.0f);
-    if (FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(GPS_RESCUE_MODE)) {
+    if (FLIGHT_MODE(ANGLE_MODE|GPS_RESCUE_MODE|ALTHOLD_MODE)) {
```

Comment:
I would love to have some ACRO + ALTHOLD mode, but I'm afraid we need more logic for it to prevent corner cases - eg flips / flying upside down / etc. Not sure if it's a good idea to allow users to mix ALTHOLD with any mode on their wish - we can't guarantee that everything works good their.